config/locales/en.yml

```diff
@@ -85,7 +85,7 @@ en:
         name: "Art and design"
         group: "Secondary"
         sub_head: "Art and design - Secondary"
-        funding: "Bursaries of £10,000 are available for trainee art and design teachers if you’re eligible (non-UK citizens without indefinite leave to remain in the UK are unlikely to be eligible)."
+        funding: "Bursaries of %{widget_subject_art_and_design_bursary} are available for trainee art and design teachers if you’re eligible (non-UK citizens without indefinite leave to remain in the UK are unlikely to be eligible). Also inserting another variable as a test, our AO low value: %{fees_assessment_only_low} and one without any hyphens in, our low iQTS fee: %{fees_iQTS_low}."
```

Comment:
Based on my other comment, this will be `%{widget_subject_art__and__design__bursary`